Sources/Shared/Request+Root.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  Request+Root.swift
```

Comment:
This extension is re: Folders so should be named as such. Root is only one request related to folders and is too specific. File should be renamed as well.

```suggestion
//  Request+Folders.swift
```